packages/relay/src/formatters.ts

```diff
@@ -38,7 +38,10 @@ const generateRandomHex = (bytesLength = 16) => {
  * Format message prefix for logger.
  */
 const formatRequestIdMessage = (requestId?: string): string => {
-  return requestId ? `[${constants.REQUEST_ID_STRING}${requestId}]` : '';
+  if (!requestId) {
+    return '';
+  }
+  return requestId.includes(constants.REQUEST_ID_STRING) ? requestId : `[${constants.REQUEST_ID_STRING}${requestId}]`;
```

Comment:
nit: We can only check the length of the string, since we know that UUID strings have a length of `36`

> A standard UUID code contains 32 hex digits along with 4 “-” symbols, which makes its length equal to 36 characters.

```suggestion
  if (requestId.length > UUID_LENGTH) {
    return requestId;
  }
  return `[${constants.REQUEST_ID_STRING}${requestId}]`;
```
```suggestion
  return requestId.length > UUID_LENGTH ? requestId : `[${constants.REQUEST_ID_STRING}${requestId}]`;
```

Reply:
Ah, I see. Thanks, that's a good point! However, I think `constants.REQUEST_ID_STRING` is already present and used. Since we know a formatted request ID will have that prefix, checking if the passed-in argument has that prefix should be sufficient enough.

Reply:
Checking only the length is slightly faster - takes `O(1)` time, whereas checking if the string contains the prefix is roughly `O(m)` (where `m` is the length of the prefix), but the difference is negligible cause our prefix is a very short string, so it's up to you!